python/cloud-init/scripts/kube-vip-cp-ha-bgp.sh

```diff
@@ -4,7 +4,7 @@ set -e
 echo "Install kube-vip for HA..."
 
 CONTROL_PLANE_IP=$(jq -r ".controlPlaneIp" /run/customdata.json)
-KUBE_VIP_VERSION="v0.5.12" #TODO $(jq -r ".kubeVipVersion" /run/customdata.json)
+KUBE_VIP_VERSION="v0.5.12" #TODO (ocobleseqx) should it be customisable $(jq -r ".kubeVipVersion" /run/customdata.json)?
```

Comment:
Again with not v0.6.0

Reply:
addressed in https://github.com/equinix-labs/pulumi-equinix-kubernetes-cluster/pull/9/commits/a0b9506847dc23d818987ed5bc592684addac079